paprika/benchmark.py

```diff
@@ -138,5 +138,5 @@ def wrapper_hotspots(*args, **kwargs):
 
 def profile(decorated_fn=None, *, n_runs=1, top_n=10):
     decorated_class = access_counter(decorated_fn)
-    decorated_class = hotspots(decorated_class)
+    decorated_class = hotspots(decorated_class, n_runs=n_runs, top_n=top_n)
```

Comment:
To silence linter and make sure the args actually get passed you can mock hotspots, but ideally you'd have a test that ensures that covers hotspots func as well.

Reply:
The main issue with the test coverage of `@hotspots` is that we can't reliably/meaningfully compare the `cProfiler` output. It's probably gonna have to be a mock indeed